pkg/agent/controller/traceflow/traceflow_controller_test.go

```diff
@@ -100,6 +102,12 @@ var (
 	}
 )
 
+type fakeNodeRouteQuerier struct{}
+
+func (f *fakeNodeRouteQuerier) LookupIPInPodSubnets(ip netip.Addr) (bool, bool) {
+	return podCIDR2IPv4.Contains(ip), false
```

Comment:
maybe define a podCIDR that includes both `pod1IPv4` and `pod2IPv4` for clarity. Something like `podCIDR = netip.MustParsePrefix("192.168.0.0/20")`

Reply:
Fixed.